src/main/java/telraam/station/FetcherFactory.java

```diff
@@ -0,0 +1,42 @@
+package telraam.station;
+
+import org.jdbi.v3.core.Jdbi;
+import telraam.database.models.Station;
+import telraam.logic.lapper.Lapper;
+import telraam.logic.positioner.Positioner;
+import telraam.station.http.HTTPFetcher;
+import telraam.station.websocket.WebsocketFetcher;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Set;
+import java.util.logging.Logger;
+
+public class FetcherFactory {
```

Comment:
:cry:

Reply:
I think a factory was better than adding a switch statement in the app.java, it removes complexity from the app.java file